test/parallel/test-eslint-inspector-check.js

```diff
@@ -12,12 +12,18 @@ const message = 'Please add a skipIfInspectorDisabled() call to allow this ' +
 new RuleTester().run('inspector-check', rule, {
   valid: [
     'foo;',
-    'common.skipIfInspectorDisabled(); require("inspector");'
+    'common.skipIfInspectorDisabled(); require("inspector");',
+    `require("common");
+     common.skipIfInspectorDisabled();
+     require("inspector");`
   ],
   invalid: [
     {
       code: 'require("inspector")',
-      errors: [{ message }]
+      errors: [{ message }],
+      output: 'require("common");\n' +
+              'common.skipIfInspectorDisabled();\n' +
+              'require("inspector");'
```

Comment:
@shobhitchittora is this definitely going to work and will it also insert `require("common");` here? Because I do not see this happening in the fixer.